<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParticipantIdentifactionForm</class>
 <widget class="QWidget" name="ParticipantIdentifactionForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>917</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_9">
   <property name="leftMargin">
    <number>50</number>
   </property>
   <property name="topMargin">
    <number>50</number>
   </property>
   <property name="rightMargin">
    <number>50</number>
   </property>
   <property name="bottomMargin">
    <number>50</number>
   </property>
   <item row="11" column="0" colspan="2">
    <widget class="QLabel" name="labelAnnotationTypes">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Si oui, quels types de phénomènes avez-vous déjà annoté ?</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayoutAge">
     <item row="0" column="0">
      <widget class="QLabel" name="labelAge">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Âge:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditAge">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="hspacerAge">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>538</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="6" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayoutL1">
     <item row="0" column="0">
      <widget class="QLabel" name="labelL1">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Langue maternelle:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="radioButtonL1French">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>français</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroupL1</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QRadioButton" name="radioButtonL1Other">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>autre :</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroupL1</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="lineEditL1">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayoutHearingProblems">
     <item row="0" column="0">
      <widget class="QLabel" name="labelHearingProblems">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Avez-vous des problèmes auditifs déclarés?</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="radioButtonHearingProbYes">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>oui</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroupHearingProblems</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QRadioButton" name="radioButtonHearingProbNo">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>non</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroupHearingProblems</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="3">
      <spacer name="hspacerHearingProblems">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="labelIdentification">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Identification de participant</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <layout class="QGridLayout" name="gridLayoutAnnotationExperience">
     <item row="0" column="0">
      <widget class="QRadioButton" name="radioButtonAnnotExperienceYes">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>oui</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroupAnnotationExperience</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="radioButtonAnnotExperienceNo">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>non</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroupAnnotationExperience</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="12" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayoutAnnotationTypes">
     <item row="2" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEditAnnotOther">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="checkBoxAnnotOther">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Autres :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QCheckBox" name="checkBoxAnnotIntonation">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Intonation</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QCheckBox" name="checkBoxAnnotAccentuation">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Accentuation</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QCheckBox" name="checkBoxAnnotProminence">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Proéminence</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QCheckBox" name="checkBoxAnnotProsodicUnits">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Unités prosodiques</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="17" column="0" colspan="2">
    <spacer name="verticalSpacerBottom">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayoutMusicalTraining">
     <item row="0" column="0">
      <widget class="QLabel" name="labelMusicalTraining">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Avez-vous une pratique musicale?</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="radioButtonMusicalTrainingYes">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>oui</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroupMusicalTraining</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QRadioButton" name="radioButtonMusicalTrainingNo">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>non</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroupMusicalTraining</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="3">
      <spacer name="hspacerMusicalTraining">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="5" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayoutEmail">
     <item row="0" column="0">
      <widget class="QLabel" name="labelEmail">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Adresse e-mail:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditEmail">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QLabel" name="labelAnnotationExperience">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string> Avez-vous déjà annoté la prosodie dans des corpus oraux en français ?</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <spacer name="verticalSpacerTop">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="15" column="0" colspan="2">
    <spacer name="verticalSpacerBeforeButton">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayoutSex">
     <item row="0" column="0">
      <widget class="QLabel" name="labelSex">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Sexe:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="radioButtonSexMale">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>homme</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroupSex</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QRadioButton" name="radioButtonSexFemale">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>femme</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroupSex</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="3">
      <spacer name="hspacerSex">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>328</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <spacer name="verticalSpacerBeforeQuestions">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="labelTheoreticalBackground">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Sur quel type d’approche vos annotations étaient-elles principalement basées ?</string>
     </property>
    </widget>
   </item>
   <item row="16" column="0" colspan="2">
    <widget class="QPushButton" name="buttonFinished">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Cliquez ici pour commencer l'expérience</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QLineEdit" name="lineEditBackground">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditAge</tabstop>
  <tabstop>radioButtonSexMale</tabstop>
  <tabstop>radioButtonSexFemale</tabstop>
  <tabstop>lineEditEmail</tabstop>
  <tabstop>radioButtonL1French</tabstop>
  <tabstop>radioButtonL1Other</tabstop>
  <tabstop>lineEditL1</tabstop>
  <tabstop>radioButtonHearingProbYes</tabstop>
  <tabstop>radioButtonHearingProbNo</tabstop>
  <tabstop>radioButtonMusicalTrainingYes</tabstop>
  <tabstop>radioButtonMusicalTrainingNo</tabstop>
  <tabstop>radioButtonAnnotExperienceYes</tabstop>
  <tabstop>radioButtonAnnotExperienceNo</tabstop>
  <tabstop>checkBoxAnnotAccentuation</tabstop>
  <tabstop>checkBoxAnnotProminence</tabstop>
  <tabstop>checkBoxAnnotIntonation</tabstop>
  <tabstop>checkBoxAnnotProsodicUnits</tabstop>
  <tabstop>checkBoxAnnotOther</tabstop>
  <tabstop>lineEditAnnotOther</tabstop>
  <tabstop>lineEditBackground</tabstop>
  <tabstop>buttonFinished</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroupL1"/>
  <buttongroup name="buttonGroupSex"/>
  <buttongroup name="buttonGroupHearingProblems"/>
  <buttongroup name="buttonGroupMusicalTraining"/>
  <buttongroup name="buttonGroupAnnotationExperience"/>
 </buttongroups>
</ui>
